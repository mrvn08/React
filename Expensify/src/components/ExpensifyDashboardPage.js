import React from 'react';

const ExpenseDashboardPage = () => (
    <div>
        This is a dashboard component
    </div>
);

export default ExpenseDashboardPage;
