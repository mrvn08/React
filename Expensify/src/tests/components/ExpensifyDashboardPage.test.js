import React from 'react';
import { shallow } from 'enzyme';
import ExpensifyDashboardPage from '../../components/ExpensifyDashboardPage'

test('Should render dashboard page properly', () => {
    const wrapper = shallow(<ExpensifyDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});
