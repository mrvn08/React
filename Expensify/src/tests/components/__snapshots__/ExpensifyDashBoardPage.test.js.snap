// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render dashboard page properly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
